### ISSUE TITLE
Documention for 0.2

### ISSUE THREAD

**Issue by miina** [xwp/pwa-wp#68]
Add documentation for developers for version 0.2, including:
- Update the existing documentation according to changes;
- Using custom offline / error templates;
- Actions and filters;

**Comment by postphotos** [comment]
Thanks @miina for adding this issue! I've fleshed out a basic user story. I'm assuming the docs will go in this project's Wiki, as AMP did before the product site. Thoughts? 

cc: @westonruter

**Comment by amedina** [comment]
Correct. Documentation at this point should be in the Wiki. Should provide a better logical organization os the Wiki, as compared to the AMP plugin wiki.

**Comment by westonruter** [comment]
Yeah, at the moment the readme has the docs. But this should move to the wiki to make it easier to maintain.

**Comment by postphotos** [comment]
A minor update here - As we build out #90, we'll be adding to this set of documentation to showcase the features of the PWA plugin.

**Comment by alcurrie** [comment]
_Updated list of Features in the PWA plugin_ (From @miina )
- **Turbo Mode** (Precaching all admin assets by default) 
- **Streaming** (Displaying the header in case of requests taking longer time even if the site hasn’t actually been loaded + showing a streaming bar) 
- **Default offline and 500 templates** 
- **Override offline and 500 templates with custom templates** -  In Progress: (https://github.com/westonruter/twentyseventeen-westonson/issues/4) 
- **Offline commenting**  -  In Progress:  (https://github.com/westonruter/twentyseventeen-westonson/issues/3) 
- **Detect HTTPS and add UI for enforcing that** 
- **Allow registering Service Workers by multiple plugins using the API** (displays console warning if case of conflicting routes)
